jest.ssr.config.js

```diff
@@ -34,7 +34,7 @@ module.exports = {
     'packages/'
   ],
 
-  testTimeout: 30000,
+  testTimeout: 50000,
```

Comment:
oof, was it the SSR tests you mentioned that prompted this?

Reply:
yeah, I'm not sure why they take longer now

Reply:
attempting to fix with maxConcurrency instead, might just be something with how node does concurrency now